files/image_config/interfaces/interfaces.j2

```diff
@@ -5,6 +5,11 @@
 # file: /etc/network/interfaces
 #
 {% endblock banner %}
+{% if (MGMT_VRF_CONFIG) and (MGMT_VRF_CONFIG['vrf_global']['mgmtVrfEnabled'] == "true") %}
+auto mgmt
+iface mgmt
+    vrf-table 5000
```

Comment:
Isn't the table in the other PR 4999?

Reply:
they need to modify the other pr accordingly.

Reply:
> Isn't the table in the other PR 4999?

@nikos-github It's rebased now and shows 5000.